code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -5,11 +5,11 @@
 /*
 replicate
 	-weight 7
-	-precondition: can_afford(FLOCK_LAY_EGG_COST) and less than FLOCK_DRONE_LIMIT drones
+	-precondition: not in tutorial, can_afford(FLOCK_LAY_EGG_COST), and less than FLOCK_DRONE_LIMIT drones
 
 nest
 	-weight 6
-	-precondition: can_afford(FLOCK_LAY_EGG_COST) and less than FLOCK_DRONE_LIMIT drones
```

Comment:
Comments needed below too at REPLICATION GOAL and NEST + REPLICATION GOAL sections

Reply:
guh why do we duplicate information 3 times